Update ome-advanced-message-encryption.md

lindabrGithub opened the pull request:
@KCCross we have request from EEE to add the following note due to PG escalation.  Can you help with this.

"Advanced Message Encryption revocation and expiration functions will not function unless the "Premium Encryption in Office 365" option is enabled in their E5 license."

lindabrGithub committed:
Update ome-advanced-message-encryption.md

@KCCross we have request from EEE to add the following note due to PG escalation.  Can you help with this.

"Advanced Message Encryption revocation and expiration functions will not function unless the "Premium Encryption in Office 365" option is enabled in their E5 license."

denisebmsft committed:
Merge branch 'public' into patch-10

JohanFreelancer9 commented on microsoft-365/compliance/ome-advanced-message-encryption.md:
```diff
@@ -26,6 +26,8 @@ Advanced Message Encryption helps customers meet compliance obligations that req
 
 You can only revoke and set an expiration date for emails sent to external recipients.
 
+Note:"Advanced Message Encryption revocation and expiration functions will not function unless the "Premium Encryption in Office 365" option is enabled in their E5 license."
```
```suggestion
> [!NOTE]
> The Advanced Message Encryption revocation and expiration functions will not work unless you enable the **Premium Encryption in Office 365** option in your E5 license.
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @lindabrGithub, kindly commit the suggested changes if you agree with them. Many thanks.

yogkumgit commented:
@lindabrGithub Please implement the suggested changes. Thanks

yogkumgit commented:
@lindabrGithub Gentle reminder

lindabrGithub committed:
Update microsoft-365/compliance/ome-advanced-message-encryption.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

lindabrGithub commented:
@yogkumgit , sorry for the delay, I missed the comment.  Let me know if there is anything else you need me to do to complete this.

yogkumgit commented:
@KCCross Please merge this PR. Thanks

Commit:
Merge branch 'public' into patch-10

Commit:
Changed wording

Removed from Note to primary content. This is not subsidiary information. Changed wording from negative to positive. Added the content to the E5 area of the SKU line up.
